apps/website/.vuepress/theme/global-components/ComponentsOverview.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div>
+    <ItemOverviewList :items="componentList" />
```

Comment:
It looks like this component only exists to pass the list down to the ItemOverview component. Does it make sense to move this up one level and only have the top level overview / list ad the list item template?

Reply:
I think it is a great fit in case we would like to change something when we get the list of the component before passing them to the `ItemOverviewList`

Reply:
After playing around a little more with setting the footer for each component I believe the overview list is not a great fit. I already started to move some of the code into `ComponentOverview` component. Will push soon.